VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxLocalParseChannel.java

```diff
@@ -0,0 +1,45 @@
+package org.edx.mobile.module.notification;
+
+import java.io.Serializable;
+
+/**
+ * need a place to hold the logic of mapping courseId to channelId.
+ */
+public class EdxLocalParseChannel implements Serializable{
```

Comment:
not sure putting "edx" in the class name is super useful since it's already implied by the package?